benchmark/docker-compose.yml

```diff
@@ -84,7 +83,6 @@ services:
     - --rpc-ws-enabled
     - --rpc-ws-apis=admin,eth,miner,web3,net,priv,eea
     - --rpc-ws-host=0.0.0.0
-    - --host-whitelist=*
```

Comment:
instead of deleting this line, can you change to host-allowlist

Reply:
Changed ad well, thanks for reviewing!